### PR TITLE
Removed tno services from deployment

### PR DESCRIPTION
**Pull request by laidaoyu**
- removed content-current-service and contentmigration-recent-service

### CODE REVIEW

**Review by JacobWang-bc — APPROVED** [review]
looks good for me

File: openshift/kustomize/tekton/base/tasks/deploy-all.yaml [hunk 1]
```diff
@@ -117,16 +117,6 @@ spec:
           [env]="dev test prod"
         )
 
-        declare -A COMPONENT5=(
```
Comment (Fosol) [inline openshift/kustomize/tekton/base/tasks/deploy-all.yaml:120]: You will need to resync these values due to the way shell scripts work.

**Review by Fosol — CHANGES_REQUESTED** [review]

**Comment by Fosol** [comment]
You will also need to update the deployment scripts for deployment

File: openshift/kustomize/tekton/base/tasks/deploy-all.yaml (hunk 1 above)
Comment (laidaoyu) [reply openshift/kustomize/tekton/base/tasks/deploy-all.yaml:120]: I have updated the script. thanks!

**Review by nehalaggarwal-bcgov — COMMENTED** [review]
Looks good

**Pull request merged**